indico/modules/events/abstracts/templates/management/overview.html

```diff
@@ -1,118 +1,157 @@
 {% extends 'events/abstracts/management/_base.html' %}
 
 {% block content %}
-    {% if not cfa.has_started %}
-        {% include 'events/abstracts/management/cfa_actions/not_started.html' %}
-    {% elif cfa.has_started and not cfa.has_ended %}
-        {% include 'events/abstracts/management/cfa_actions/open.html' %}
-    {% elif cfa.has_ended %}
-        {% include 'events/abstracts/management/cfa_actions/ended.html' %}
-    {% endif %}
-
-    <div class="action-box">
-        <div class="section">
-            <div class="icon icon-upload"></div>
-            <div class="text">
-                <div class="label">
-                    {% trans %}Submission{% endtrans %}
-                </div>
-                {% trans %}Configure the submission dates, the abstract fields, etc.{% endtrans %}
-            </div>
-            <div class="toolbar">
-                <div class="group">
-                    <a href="#" class="i-button icon-mail"
-                       data-href="{{ url_for('.email_tpl_list', event) }}"
-                       data-title="{% trans %}Configure e-mail notifications{% endtrans %}"
-                       data-ajax-dialog>
-                        {% trans %}Notifications{% endtrans %}
-                    </a>
+    {% if not event.has_feature('abstracts') %}
```

Comment:
Why not use a separate template? There is almost no common code between the two cases.